src/App.js

```diff
@@ -34,6 +34,15 @@ function App() {
             });
     }
 
+    // function findTypeNameFromID(ID) {
```

Comment:
we don't need this commented out function anymore now that the backend is sending us the data we want